radio/src/opentx.cpp

```diff
@@ -212,6 +212,13 @@ void per10ms()
 
   outputTelemetryBuffer.per10ms();
 
+//OW
+#if defined(MAVLINK_TELEM)
+  mavlinkTelem.wakeup();
```

Comment:
we are in an interruption, is it really needed?

Reply:
now we are getting to what I consider the real beef, it's a major one of the pain points I was mentioning.

clearly, somewhere the mavlink things needs to be spun. When I started I was evaluating the possibilities as much as I could see them - but pl note that my understanding of OpenTx is limited such that I can follow patterns which I see somewhere else in the code. I could see about 3 approaches to do that, and what you see here is the one which I have finally chosen.  

**HOWEVER**; I have no idea if this is the proper or best approach !!!!

The only justification for what I did here is that it looked reasonable to me and works for me, but this doesn't mean anything. It's one of the key code points, and here one would really need you guy's insight. IMHO all the really key points come into play here, from time scheduling to resource consumption. But I am not in a position to make competent comments on any.

So, this is probably the single most important code line and IMHO would need most of your attention if it's good or bad, or what the alternatives would be.

pl advice

Reply:
See mixerTask in tasks.cpp, that's where all the wakeups are done

Reply:
YES! This is one of the other options I've explored, and to not forget about it I left in comments, see https://github.com/opentx/opentx/blob/ce97dd043ac3db79b8db083c925a4812b7da469b/radio/src/tasks.cpp#L123-L127 and https://github.com/opentx/opentx/blob/ce97dd043ac3db79b8db083c925a4812b7da469b/radio/src/targets/horus/board.cpp#L216-L220.
For reasons I cannot remember I took the other approach you see currently. I think I considered it more streamlined with other telemetry handling.
 
Is this what you are saying, that going via tasks.cpp is the correct (best?) approach?

I like to add that there are two features I wanted to do but could not yet solve:
- I would like to have an option to store the mavlink telemetry stream onto the SD card, and I've spend quite some time on this but whatever I did I failed (crashes).
- currently what is working is what I call the mavsdk, but I also would want to add a more basic mavlink so that any mavlink message can be digested in the lua scripts. This is a bit along what had been tried in an earlier PRs. I however had issues with memory. 

 Would these be working when going via tasks.cpp?